src/custom/components/SideBanner/index.tsx

```diff
@@ -0,0 +1,172 @@
+import { useState } from 'react'
+import styled from 'styled-components/macro'
+import AnniversaryImage from 'assets/cow-swap/anniversary-icons.png'
+import TwitterImage from 'assets/cow-swap/twitter.svg'
+import ReactConfetti from 'react-confetti'
+import { transparentize } from 'polished'
+import SVG from 'react-inlinesvg'
+import { CloseIcon, ExternalLink } from 'theme'
+
+const WIDTH = 440
+const HEIGHT = 440
+const ANNIVERSARY_TWEET_TEMPLATE =
+  'Holy CoW, today @MEVprotection is 1 year old! Check out their UI growth over time to see the CoW-evolution https://youtu.be/nxU11DmBVMk'
+
+export interface BannerProps {
+  type: string
+}
+
+const Banner = styled.div<{ isActive: boolean }>`
+  position: fixed;
+  width: ${`${WIDTH}px`};
```

Comment:
no need for double template strings
```suggestion
  width: ${WIDTH}px`;
```